### PR TITLE
Moved browserify to dev-dependencies and added .npmignore

### LINKED ISSUE #12: Use browserify as a dev-dependency
browser-run is the perfect tool to launch tests in a continuous integration environment, like Jenkins and Travis. We could save time, bandwidth and disk space (halt of the size of the installed package, 22 of 45 MB, corresponds to browserify) if instead of requiring browserify as a runtime dependency to bundle the `browser/reporter.js` file, it would be included that way in the published package.

I also think the package `utf8-stream` is only used in that file, so it could also be installed as a devDependency only.

Example package.json:

```json
{
  "scripts": {
    "prepublish": "browserify browser/reporter.js -o dist/reporter.js"
  },
  "devDependencies": {
    "browserify": "~5.9.3",
    "utf8-stream": "0.0.0"
  }
}
```

If you agree I can create a PR ;)

### PR DESCRIPTION
**Pull request by rubennorte**
Fixes #12

### CODE REVIEW

File: .npmignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+browser/
+script/
```
Comment (juliangruber) [inline .npmignore]: we need this folder for phantomjs

File: package.json [hunk 2]
```diff
@@ -12,22 +12,25 @@
     "browser-run": "./bin/bin.js"
   },
   "scripts": {
+    "build-reporter": "browserify browser/reporter.js -o static/reporter.js",
```
Comment (juliangruber) [inline package.json]: can you rename this to `build`?

**Comment by juliangruber** [comment]
oherwise lgtm!

File: .npmignore (hunk 1 above)
Comment (rubennorte) [inline .npmignore]: OK, I'm preserving it then. I couldn't fine a reference to it in the rest of the code. How is it linked?

File: package.json (hunk 2 above)
Comment (rubennorte) [inline package.json]: Sure

**Comment by juliangruber** [comment]
thanks! published as 2.1.0

**Comment by rubennorte** [comment]
Great! Thank you.

**Comment by ainthek** [comment]
however, having browserify as devDependency now raises this with help command:

browser-run --help
module.js:338
    throw err;
          ^
Error: Cannot find module 'browserify'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (c:\Users\marcus\AppData\Roaming\npm\node_modules\browser-run\index.js:5:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)

**Comment by ainthek** [comment]
CLI help not working withou devDependencies now:

browser-run --help
module.js:338
    throw err;
          ^
Error: Cannot find module 'browserify'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (c:\Users...\AppData\Roaming\npm\node_modules\browser-run\index.js:5:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)

**Comment by juliangruber** [comment]
moved back to deps and published as 2.1.1

**Pull request merged**